app/jobs/shipit/merge_pull_requests_job.rb

```diff
@@ -13,7 +13,14 @@ def perform(stack)
       return false unless stack.allows_merges?
 
       pull_requests.select(&:pending?).each do |pull_request|
-        return false unless pull_request.merge!
+        # Any merged PR might conflict with others, so we need to refresh.
+        # If the PR is unchanged, we should get a cache hit anyway.
+        pull_request.refresh!
+        unless pull_request.merge!
+          # That PR isn't mergeable yet, we'll try again a bit later.
+          MergePullRequestsJob.set(wait: 10.seconds).perform_later(stack)
```

Comment:
Hmmm, yeah, the boolean return values are kind of confusing. I would personally expect `true` to mean merged, `false` to be rejected, and anything else to be an exception, or not use boolean values. /shrug

If this works, it's fine to leave it like this though.